README.md

```diff
@@ -68,6 +68,28 @@ steps:
               secret-id: 'arn:aws:secretsmanager:ap-southeast-2:1234567:secret:my-global-file-secret'
 ```
 
+## For Secrets in Another Region
+
+In order to request a secret from another region from your agents, a region parameter must be provided.
+This is independent of the secret ARN, which may confusingly include a region.
```

Comment:
> which may confusingly include a region

The way this is worded makes it seem you could specify a different region to that contained in the ARN.  Would that even be possible to have as parameters and still return a valid value?

Reply:
That's a very good point, and makes this rather error prone. I'll update the PR to infer the region from the ARN.